.github/workflows/ci.yml

```diff
@@ -0,0 +1,37 @@
+name: Pull request
+on:
+  pull_request:
+    types: [opened, closed, reopened, ready_for_review, review_requested]
+
+jobs:
+
+  notifications:
+    name: Telegram notifications
+    runs-on: ubuntu-latest
+    steps:
+      - name: Send telegram message on pull request to review
+        if: github.event.pull_request.merged == false && github.event.action != 'closed'
+        uses: appleboy/telegram-action@master
```

Comment:
How about freezing version with commit hash? This approach results in less surprises or damages by malware.

Reply:
Sorry, I don't follow your suggestion. Are you suggesting changing the preconditions to sending the notification? Are you suggesting using the commit hash to trigger it? If that the case, anyone how pushes a commit on any PR will also trigger the notification.
Maybe I misunderstood you, could you extend on your suggestion?

Reply:
I was suggesting using `@<commit-hash>`  instead of `@master` (dynamic tip of branch). Perhaps it's not so critical on OSS projects, but always a potential vector.

Reply:
You are right. I changed it to use the commit hash related to the current latest tag for v.0.1.1.